app/views/idv/doc_auth/welcome.html.erb

```diff
@@ -30,78 +30,64 @@
+          <%= step += 1 %>
         </div>
-        <br/>
-        <div>
-          <%= t('doc_auth.instructions.text1a') %>
+        <div class='margin-right-1 inline-block'>
+          <div class='h1 inline-block bold'>
+            <%= t('doc_auth.instructions.bullet1a') %>
+          </div>
+          <br/>
+          <div>
+            <%= t('doc_auth.instructions.text1a') %>
+          </div>
         </div>
-      </div>
-    </li>
+      </li>
     <% end %>
     <li class='padding-top-2 padding-bottom-1'>
-      <div class='inline-block margin-right-2 margin-top-1 text-top circle circle-number bg-blue white'>
-        <%= step += 1 %>
-      </div>
-      <div class='margin-right-1 inline-block'>
-        <div class='h1 inline-block bold'>
-          <%= t('doc_auth.instructions.bullet2') %>
+      <div class="grid-row">
+        <div class='inline-block margin-right-2 margin-top-1 text-top circle circle-number bg-blue white'>
+          <%= step += 1 %>
         </div>
-        <br/>
-        <div>
-          <%= t('doc_auth.instructions.text2') %>
+        <div class='grid-col-10'>
+          <div class='h1 inline-block bold'>
+            <%= t('doc_auth.instructions.bullet2') %>
+          </div>
+          <br/>
+          <div>
+            <%= t('doc_auth.instructions.text2')  %>
+          </div>
         </div>
       </div>
     </li>
     <li class='padding-top-2 padding-bottom-1'>
-      <div class='inline-block margin-right-2 margin-top-1 text-top circle circle-number bg-blue white'>
-        <%= step += 1 %>
-      </div>
-      <div class='margin-right-1 inline-block'>
-        <div class='h1 inline-block bold'>
-          <%= t('doc_auth.instructions.bullet3') %>
+      <div class="grid-row">
+        <div class='grid-col-2 margin-right-2 margin-top-1 text-top circle circle-number bg-blue white'>
+          <%= step += 1 %>
         </div>
-        <br/>
-        <div>
-          <%= t('doc_auth.instructions.text3') %>
-        </div>
-      </div>
-    </li>
-    <li class='padding-top-2 padding-bottom-1'>
-      <div class='inline-block margin-right-2 margin-top-1 text-top circle circle-number bg-blue white'>
-        <%= step += 1 %>
-      </div>
-      <div class='margin-right-1 inline-block'>
-        <div class='h1 inline-block bold'>
-          <%= t('doc_auth.instructions.bullet4') %>
-        </div>
-        <br/>
-        <div>
-          <%= t('doc_auth.instructions.text4') %>
+        <div class='grid-col-10'>
+          <div class='h1 inline-block bold'>
+            <%= t('doc_auth.instructions.bullet3') %>
+          </div>
+          <br/>
+          <div>
+            <%= t('doc_auth.instructions.text3') %>
+          </div>
         </div>
       </div>
     </li>
   </ul>
 
+  <h2><%= t('doc_auth.instructions.privacy') %></h2>
+  <p><%== t('doc_auth.info.privacy_html', link: new_window_link_to(t('doc_auth.instructions.learn_more'),
+            'https://login.gov/policy/')) %></p>
```

Comment:
Looks like this just moved from elsewhere , but can we use our method for this URL?
```suggestion
            MarketingSite.security_and_privacy_practices_url)) %></p>
```